src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -27,21 +37,30 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        if (!isset($config['cropping_views'])) {
+            $config['cropping_views'] = self::DEFAULT_CROPPING_CONFIG;
```

Comment:
Why don't we set this value as the default value in the configuration class? Would be more consistent and more the "symfony way"